entrypoint.sh

```diff
@@ -7,6 +7,18 @@ echo "Arguments: $@"
 echo
 echo "Supported Arguments:"
 echo "   <srcdir> dir containing source (defaults to /pkg)"
+echo
+echo "Supported Environment Variables:"
+echo "   PUBLISH_TO_REPOSITORY [${PUBLISH_TO_REPOSITORY}]"
+echo "      Packagecloud repository to push packages to."
+echo "      Can be 'test', 'dev' or empty (default)"
+echo "      for not publishing them at all."
+echo "      Targetting 'test' also creates a debian tag"
+echo "      and pushes changes to determined git remote"
+echo "   PUSH_TO_GIT_REMOTE [${PUSH_TO_GIT_REMOTE}]"
```

Comment:
* @strk , do you really think we need this variable ?

Reply:
Yes, for local runs. Otherwise pushing to origin fails due to lack of the ssh key. With this variable you can, locally, specify a remote URL which includes the autorization token...

Reply:
We're not going to use it with github-action (although I left support in github-action too in case we find a simpler way to configure the authenticaiton token, right now I'm using `git remote set-url` to change the url to origin to include the token...

Reply:
OK